scripts/pre_commit_linter.py

```diff
@@ -2414,6 +2419,50 @@ def _check_extra_js_files(self):
 
         return summary_messages
 
+    def _check_constants_declaration(self):
```

Comment:
This is a good check, but I thought the point of the comment thread was to have a check to ensure that no constant is declared more than once in the codebase?

Reply:
@YashJipkate please respond to all inline comments.

Reply:
@YashJipkate Repeat: Please respond to all inline comments.

Reply:
This concern is already addressed in the later commits. This somehow missed my radar.